forch/faucetizer.py

```diff
@@ -141,11 +144,24 @@ def _augment_include_file_name(self, file_name):
         base_file_name, ext = os.path.splitext(file_name)
         return base_file_name + INCLUDE_FILE_SUFFIX + ext
 
-    def _is_access_port(self, port_cfg):
+    def _get_port_type(self, port_cfg):
+        testing_port_identifier = (self._config.orch_testing_config.testing_port_identifier or
+                                   TESTING_PORT_IDENTIFIER_DEFAULT)
+        if testing_port_identifier in port_cfg.get('description'):
+            return 'testing'
```

Comment:
create named constants for these (e.g. PORT_TYPE.TESTING)

Reply:
Added PortType enum proto in shared_constants.proto